src/components/Displays/DisplayAll.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import uuid from 'react-uuid';
+
+const DisplayAll = () => {
+  const [allReviews, setAllReviews] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get('https://martinswdev-be-nc-games.herokuapp.com/api/reviews')
+      .then(({ data }) => {
+        console.log(data.reviews);
+        setAllReviews(data.reviews);
+      });
+  }, []);
+
+  return (
+    <main key={uuid()} className="displayAll-main">
+      {allReviews.map((item) => {
+        return (
+          <div key={uuid()} className="displayAll-review">
+            <img
+              key={uuid()}
+              src={item.review_img_url}
+              alt={item.title}
+              className="displayAll-review__img"
+            />
+            <h2 key={uuid()}>{item.title}</h2>
```

Comment:
I don't think you need all the keys and uuiids. Just one will do

Reply:
Was having trouble when they were lists, will remove them and see what errors show up